di-ipv-credential-issuer-stub/src/test/java/uk/gov/di/ipv/stub/cred/handlers/AuthorizeHandlerTest.java

```diff
@@ -346,8 +346,10 @@ void generateResponseShouldCallDoAuthorizeMethodWhenInvalidJsonPayloadProvided()
 
         authorizeHandler.generateResponse.handle(mockRequest, mockResponse);
 
-        verify(mockRequest).attribute("error", "Invalid JSON");
-        verify(mockViewHelper).render(anyMap(), eq("authorize.mustache"));
+        verify(mockResponse)
+                .redirect(
+                        TEST_REDIRECT_URI
+                                + "?error=invalid_json&iss=Credential+Issuer+Stub&error_description=Unable+to+generate+valid+JSON+Payload");
```

Comment:
@Danomanic Is this the format IPV Core will expect an error payload from a CRI? (cc @merlinc @MattBidewell )